package/.pylintrc

```diff
@@ -199,7 +199,6 @@ enable=abstract-class-instantiated,
        print-statement,
        xrange-builtin,
        zip-builtin-not-iterating,
-       map-builtin-not-iterating,
```

Comment:
Just make the list somewhere in the module namespace, you keep reusing it anyway